### DISCUSSION TITLE
Ability to hide certain apps from taskbar

### DISCUSSION THREAD

**Discussion by dene14** [linuxmint/discussions#401] [Ideas]
Hello! I'd like to ask for a feature that would allow to hide certain applications from the taskbar.
When using multiple monitors (4 in my case) it's usual that I have too many "resident" apps sitting in my taskbar even though I usually invoke them through the system tray.
Some applications implement this in their settings but this is very rare feature, so would be cool if there is central place in cinnamon where I can define what apps I'd like to hide from taskbar.
As an example of such applications you can consider all the ones available through XApp Status Applet

**Comment by Jopp-gh** [comment]
A Screen-aware and workspace-aware -panel would be great indeed.
